packages/discord.js/src/structures/ActionRow.js

```diff
@@ -1,13 +1,11 @@
 'use strict';
 
 const { ActionRow: BuildersActionRow } = require('@discordjs/builders');
-const Components = require('../util/Components');
+const snakecase = require('snakecase-keys');
```

Comment:
Won't this clash with the other PR that handles snakecase conversions using lodash.snakecase?

Reply:
Yeah this will be removed